Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```diff
@@ -49,7 +49,7 @@
       - id: RubberStampWarden
       - id: DoorRemoteArmory
       - id: HoloprojectorSecurity
-      - id: ClothingEyesHudSecurity
+      - id: ClothingEyesHudSecurity #Delta-V: Sechud Scarcity 
```

Comment:
So what exactly is getting changed besides the comment?

Reply:
Originally, I had it removed. It ended up causing some issues and I'm currently working on trying to fix it.